addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml

```diff
@@ -4,17 +4,17 @@
     <t t-name="mail.ActivityMarkDonePopover" owl="1">
         <div class="o_ActivityMarkDonePopover" t-attf-class="{{ className }}" t-on-keydown="activityMarkDonePopoverView and activityMarkDonePopoverView.onKeydown" t-ref="root">
             <t t-if="activityMarkDonePopoverView">
-                <textarea class="form-control o_ActivityMarkDonePopover_feedback" rows="3" placeholder="Write Feedback" t-on-blur="activityMarkDonePopoverView.onBlur" t-ref="feedbackTextarea"/>
-                <div class="o_ActivityMarkDonePopover_buttons">
+                <textarea class="form-control o_ActivityMarkDonePopover_feedback mb-3" rows="3" placeholder="Write Feedback" t-on-blur="activityMarkDonePopoverView.onBlur" t-ref="feedbackTextarea"/>
+                <div class="o_ActivityMarkDonePopover_buttons mb-2">
```

Comment:
Typo: ~~mb-3~~ -> `mb-2`

For the rest, when elements are organised in a column, is good practice to use `margin-bottom` rather than `margin-top`.

Reply:
Did you make the change? The whole box still has more spacing top/bottom (12px) than left/right (9px).

Reply:
![image](https://user-images.githubusercontent.com/38213504/151351614-876a7489-6618-4dcc-b963-369aaf7647ab.png)

Reply:
The one that I don't like the most is the 12px at bottom which I feel is too much. That's the only thing that changed in this PR (it was less before).

Reply:
There are no specific guidelines when it comes to "form-popover" content spacing.

My guess is that this kind of components should look like "small modals", so font-size and margins/paddings should be **tinier** but **proportional** to modal values. Here you have a comparison on master

![image](https://user-images.githubusercontent.com/8558351/151363604-3fa91128-2ad4-47b2-9336-b856788918c2.png)

As you can see dialog's buttons have a much wider spacing compared to the mark-done's ones.
Choosing between a "pixel-perfect arrangement" or consistency, I'd go for the second one.

Btw, I suggest to don't overthink that, brd is reviewing  popover component in this exact moment, chances are that we'll have to review this anyway later.

Reply:
But the modal in your example looks good because both left/right and top/bottom have the same spacing.

The inconsistency introduced in this PR in regard to the bottom spacing makes this popover look less good than before.

I would not worry if we didn't change the value, and wanted to improve it later. But here we will me making it less good (in my opinion) until we fix it, so I'd rather if we just set a proper value directly now!

It is too small in master (6px), this PR is making it too big (12px), it can't be that hard to have the middle value (9px) instead, to match the left/right spacing?